src/software/ai/hl/stp/play/enemy_free_kick_play.cpp

```diff
@@ -62,9 +62,26 @@ void EnemyFreekickPlay::getNextTactics(TacticCoroutine::push_type &yield,
         // Add Freekick shadower tactics
         tactics_to_run[0].emplace_back(shadow_free_kicker_1);
         tactics_to_run[0].emplace_back(shadow_free_kicker_2);
-        // Add Crease defender tactic
-        crease_defender_tactic->updateControlParams(world.ball().position(),
-                                                    CreaseDefenderAlignment::CENTRE);
+        // Add Crease defender tactic on side of open enemy threats
+        if (enemy_threats.size() >= 4)
+        {
+            if (enemy_threats.at(3).robot.position().y() > 0)
```

Comment:
Can you explain this part? Why do we look at that particular enemy threat and how do we decide whether or not to crease left or right?

Reply:
so the first 3 enemy threats are covered by shadow free kicker and shadow enemy so we lean the crease defender on the side with the next highest threat so that it's slightly closer to a possible shot. It also didn't seem like having a centred crease defender was very useful as it would block the same thing the goalie would block.